### PR TITLE
Removed ETH signature schema verification

### PR DESCRIPTION
**Pull request by juandavidkincaid**
[LE-236] (https://constellationnetwork.atlassian.net/browse/LE-236)

Favoring compatibility with MetaMask's `personal_sign` method.

### Changes
+ Removed schema verification for ETH signing.
+ Removed metadata title when there is no metadata.

[LE-236]: https://constellationnetwork.atlassian.net/browse/LE-236?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: source/scenes/external/SignatureRequest/index.tsx [hunk 1]
```diff
@@ -38,10 +39,10 @@ const SignatureRequest = () => {
   const { data: stringData } = queryString.parse(location.search);
 
   const { signatureRequestEncoded, asset }:
-    {  signatureRequestEncoded: string, asset: string } = JSON.parse(stringData as string);
+    { signatureRequestEncoded: string, asset: string } = JSON.parse(stringData as string);
```
Comment (AlexBrandes) [inline source/scenes/external/SignatureRequest/index.tsx:42]: This was already in the codebase but I'm just noticing now... can we put these two JSON.parse calls inside a try/catch? If the wrong format of data was sent then we'd crash the app here without the try/catch.

File: source/scenes/external/SignatureRequest/index.tsx [hunk 2]
```diff
@@ -2,7 +2,7 @@
 // Modules Imports
 /////////////////////
 
-import React, {} from 'react';
+import React, { } from 'react';
```
Comment (AlexBrandes) [inline source/scenes/external/SignatureRequest/index.tsx:5]:
```suggestion
import React from 'react';
```

**Review by AlexBrandes — APPROVED** [review]
Just a couple small comments but it looks ok otherwise. Does this mean we now have two different formats for DAG message signing and ETH message signing? Would it make sense to migrate the DAG format to match the ETH format in the future?

File: source/scenes/external/SignatureRequest/index.tsx (hunk 1 above)
Comment (juandavidkincaid) [reply source/scenes/external/SignatureRequest/index.tsx:42]: Not exactly... Verification is done in requests.ts... And this JSON.parse is from the data sent by us to the screen...

Comment (juandavidkincaid) [reply source/scenes/external/SignatureRequest/index.tsx:42]: Here...
https://github.com/StardustCollective/stargazer-wallet-ext/blob/16da0111f15005db06963dcd4cc6653836961f96/source/scripts/Background/controllers/MessageHandler/requests.ts#L54-L60

Comment (juandavidkincaid) [reply source/scenes/external/SignatureRequest/index.tsx:42]: Yeah, but I mean, those details are sent by us... we would notice while coding...

Comment (juandavidkincaid) [reply source/scenes/external/SignatureRequest/index.tsx:42]: This happens as well in other screens
https://github.com/StardustCollective/stargazer-wallet-ext/blob/e3c4b198132e68e3a8a6ca18f4616ba4a4c296b8/source/scenes/external/ApproveSpend/ApproveSpend.tsx#L63-L67

**Comment by juandavidkincaid** [comment]
### Latest Changes
+ Require the request address to be the same as the active wallet.
+ Support for hex messages signing (eth).
+ Extend RPCError from Error.
+ Add new version property to the provider, the Stargazer version.

**Pull request merged**